### DISCUSSION TITLE
Where is the code for the Undocked Deployment Extension Handler (DEH)?

### DISCUSSION THREAD

**Discussion by riverar** [microsoft/WindowsAppSDK#1710] [Q&A] [answered]
Not too familiar with DEHs so wanted to peruse around during lunch. Is the code for this checked in?

Reference: https://github.com/microsoft/WindowsAppSDK/issues/1599

**Comment by BenJKuhn** [comment, marked as answer]
The DEH component is in a closed-source repository. Deployment extensions have trust requirements that can't be satisfied outside of our internal development environment. It also dependencies on other non-public code. We're trying to open source as much as practical as quickly as practical, but this particular component is likely to remain closed source for the foreseeable future.

Ben

**Reply by riverar** [reply]
Thanks @BenJKuhn, that's fair. I'm just glad to hear I wasn't just missing it on the repo somewhere.
